<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
    <property name="spacing">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <family>Source Code Pro Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>32</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>32</number>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Максимальное число производственных участков</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="maxFeatureVectorsSpinBox">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Размер векторов-признаков</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="maxFeaturesSpinBox">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="value">
         <number>11</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Максимальное количество векторов-прототипов</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="maxPrototypeVectorsSpinBox">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>β</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="betaSpinBox">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Внимательность</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="solveButton">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Решить</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="vigilanceSpinBox">
        <property name="font">
         <font>
          <family>Source Code Pro Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
